skimage/future/graph/rag.py

```diff
@@ -293,6 +299,17 @@ def edges_iter(self, *args, **kwargs):
         except AttributeError:
             return super(RAG, self).edges(*args, **kwargs)
 
+    def subgraph(self, nbunch):
+        if self.version1:
+            return super(RAG, self).subgraph(nbunch)
+        else:
+            G = super(RAG, self).subgraph(nbunch)
```

Comment:
In v2.0 ```G.subgraph(nbunch)``` returns a view (read-only graph structure) of the original graph. 
In v1.x, it returns a new graph.  To obtain a new graph here we should probably use

    G = super(RAG, self).subgraph(nbunch).copy()